app/models/hint.rb

```diff
@@ -1,5 +1,36 @@
+require 'stringex/core_ext'
```

Comment:
You might not need be required to `require` here (ha!) as most gems auto require themselves when they are loaded from the `Gemfile`